translations/src/config.js

```diff
@@ -44,20 +58,7 @@ class Config {
     fa_pr: ['fa'],
```

Comment:
Please be patient with me if I still didn't undestand. But why `fa_pr` and not `fa-pr`?

Reply:
This are just shitty codes from the cms that could possibly occur, thats why we handle them here afaik.